### ISSUE TITLE
Using ONNX as an intermediary

### ISSUE THREAD

**Issue by modanesh** [onnx/onnx#4757]
# Ask a Question

### Question
I'm trying to load a `pth` model in Go. Can ONNX be used as an intermediary to convert a model from Pytorch's `pth` to GoLangs's `gob`?

_Originally posted by @modanesh in https://github.com/onnx/onnx/discussions/4755_

### Further information
- Relevant area: model conversion

- Is this issue related to a specific model?
Not really.

### Notes
N/A

**Comment by jcwchen** [comment]
Hi @modanesh,
Although there is no ONNX to GoLang conversion support under official [onnx org](https://github.com/orgs/onnx/repositories), I notice there might be some awesome external tool which are useful. For instance, https://github.com/owulveryck/onnx-go.

**Comment by modanesh** [comment]
Thanks @jcwchen
I tried the [ONNX-GO](https://github.com/owulveryck/onnx-go) initially. From my understanding, ONNX-GO loads the model into a computation graph and then runs it. In the ONNX-GO's repo, it's recommended to use [Gorgonia](https://github.com/gorgonia/gorgonia) as the backend, and that's what I did. However, in order to run the model, it needs to already have all the required operators of the model implemented. And for my case, there was a missing operator. I created an issue in the ONNX-GO's repo (https://github.com/owulveryck/onnx-go/issues/203), but no reply. That's when I went for the plan B, which was to use ONNX as an intermediary to convert a PyTorch's `pth` model into a GoLang's `gob`, hence the question above.
